pkg/cli/mirror/copy.go

```diff
@@ -110,70 +121,93 @@ func (o *MirrorOptions) bulkImageCopy(isc *v1alpha2.ImageSetConfiguration, srcSk
 				})
 			}
 		}
-		log.Printf("INFO: List of packages selected for copy :\n %v \n", pkgList)
-
 		for _, pkg := range pkgList {
 
-			log.Printf("INFO: collecting all related images for %s \n", pkg.Name)
+			klog.Infof("Collecting all related images for %s \n", pkg.Name)
 			for _, file := range files {
-				if strings.Contains(pkg.Name, file.Name()) {
-					// read the config.json to get releated images
-					relatedImages, err := getRelatedImages(filepath.Join(ctlgConfigsDir, file.Name()))
+				klog.V(2).Infof("File :%s\n", file.Name())
+				// read the config.json to get releated images
+				relatedImages, err := getRelatedImages(ctlgConfigsDir, []v1alpha2.IncludePackage{pkg})
+				if err != nil {
+					return err
+				}
+				for _, i := range relatedImages {
+					name := i.Name
+					if name == "" {
+						name = "bundle"
+					}
+					srcTIR, err := image.ParseReference(i.Image)
```

Comment:
Can we change this to see if `i.Image` is equal to "" and just log the invalid image and continue with the processing? We have some bundles that have incorrectly set their related images and it would be good to just ignore those, but have the logs indicate that there is a problem with that entry.

Reply:
Note this same logic should be applied to the mirror path too.

Reply:
Resolved in https://github.com/openshift/oc-mirror/pull/516/commits/62f599bf584ebba622153067b509d1ecc16893b3